### PR TITLE
issue 5463 - Clarify Hibernate ORM property setting in prod

### LINKED ISSUE #5463: Make an explicit section on Hibernate ORM in production with the different defaults
The default import.sql file is not executed when running in Native mode. According to the docs, Hibernate ORM, upon boot, will read and execute the SQL statements in the /import.sql file (if present).
That works in JVM Mode. In native mode the script is not executed. As workaround, you have to set explicitly the property:
`quarkus.hibernate-orm.sql-load-script=import.sql`
I have used this project to reproduce the issue: https://bit.ly/32Iw7Zh

Environment:
Quarkus 1.0.0.CR1

$ uname -a
Linux fedora 4.18.16-300.fc29.x86_64 #1 SMP Sat Oct 20 23:24:08 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

GraalVM: graalvm-ce-19.2.0

$GRAALVM_HOME/bin/java -version
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (build 1.8.0_222-20190711120915.buildslave.jdk8u-src-tar--b08)
OpenJDK 64-Bit GraalVM CE 19.2.0 (build 25.222-b08-jvmci-19.2-b02, mixed mode)

### PR DESCRIPTION
**Pull request by emmanuelbernard**
Fixes #5463

### CODE REVIEW

**Comment by emmanuelbernard** [comment]
for https://github.com/quarkusio/quarkus/issue/5463

File: docs/src/main/asciidoc/hibernate-orm.adoc [hunk 1]
```diff
@@ -304,6 +304,29 @@ This is great because you can define different combinations of Hibernate ORM pro
 ./mvnw compile quarkus:dev -Dquarkus.profile=dev-with-data
 --
 
+== Hibernate ORM in production mode
+
+Quarkus comes with default profiles (`dev`, `test` and `prod`).
+And you can add your own custom profiles to describe various environments (`staging`, `prod-us`, etc).
+
+The Hibernate ORM Quarkus extensions sets some defaults differently in dev and test modes than in other environments.
```
Comment (machi1990) [inline docs/src/main/asciidoc/hibernate-orm.adoc]:
```suggestion
The Hibernate ORM Quarkus extension sets some default configurations differently in dev and test modes than in other environments.
```

Comment (emmanuelbernard) [reply docs/src/main/asciidoc/hibernate-orm.adoc]: Applied, thanks

**Review by Sanne — APPROVED** [review]
nice. Thanks!

**Pull request merged**